src/client/src/app/app.module.ts

```diff
@@ -30,6 +30,7 @@ import { SidebarComponent } from './components/sidebar/sidebar.component';
 import { MaterialModule } from './material/material.module';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { MerchDetailsComponent } from './components/merch-details/merch-details.component';
+import { UsersGamesListComponent } from './modules/games/components/users-games-list/users-games-list.component';
```

Comment:
you can delete this import .. this component is being used in the GameModule

Reply:
did not see that... going to delete now